modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -4070,13 +4007,18 @@ SUBROUTINE SetVTKParameters(p_FAST, InitOutData_ED, InitOutData_AD, InitInData_S
    !.......................
 
    !bjj: interpolate here instead of each time step?
-   if ( allocated(InitOutData_SeaSt%WaveElevSeries) ) then
-      call move_alloc( InitInData_SeaSt%WaveElevXY, p_FAST%VTK_Surface%WaveElevXY )
-      call move_alloc( InitOutData_SeaSt%WaveElevSeries, p_FAST%VTK_Surface%WaveElev )
+   if ( allocated(InitOutData_SeaSt%WaveElevVisGrid) ) then
+print*,'Storing Wave surface visualization'
```

Comment:
Is this a leftover debugging statement?

Reply:
Not sure how that got left in there.

I'll make a PR to fix the issues you pointed out.  Thanks @bjonkman!

Reply:
Addressed in PR #2071